### PR TITLE
fix: clarify conversion of Wad to SignedRay

### PR DESCRIPTION
**Pull request by tserg**
This PR replaces the implementation of `Into<Wad, SignedRay>` with a helper function `wad_to_signed_ray` to better reflect the semantics that the wad value is being scaled by `10**9`, whereas the `Into` trait typically suggests that the underlying value is unmodified.

### CODE REVIEW

**Review by milancermak — APPROVED** [review]
Just to note: this is a breaking change so we should push a new release with a version bump out soon.

**Comment by tserg** [comment]
> Just to note: this is a breaking change so we should push a new release with a version bump out soon.

Good point, I will prepare `0.3.0` release once both open PRs are merged.

**Review by javra — APPROVED** [review]

**Comment by milancermak** [comment]
Can you please bump `snforge_std` to at least 0.14 with the release? Will unblock me with one issue in Opus.

**Comment by tserg** [comment]
> Can you please bump `snforge_std` to at least 0.14 with the release? Will unblock me with one issue in Opus.

Yes, I will open a separate PR for that.

**Pull request merged**
